ansible/templates/minigraph_dpg_asic.j2

```diff
@@ -204,5 +204,41 @@
       <DownstreamSummaries/>
       <DownstreamSummarySet xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution"/>
     </DeviceDataPlaneInfo>
+{% for asic in fabric_info %}
+    <DeviceDataPlaneInfo>
+      <IPSecTunnels/>
+      <LoopbackIPInterfaces xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution">
+        <a:LoopbackIPInterface>
+          <Name>HostIP</Name>
+          <AttachTo>Loopback0</AttachTo>
+          <a:Prefix xmlns:b="Microsoft.Search.Autopilot.NetMux">
+            <b:IPPrefix>{{ asic['ip_prefix'] }}</b:IPPrefix>
+          </a:Prefix>
+          <a:PrefixStr>{{ asic['ip_prefix'] }}</a:PrefixStr>
+        </a:LoopbackIPInterface>
+        <a:LoopbackIPInterface>
+          <Name>HostIP</Name>
+          <AttachTo>Loopback0</AttachTo>
+          <a:Prefix xmlns:b="Microsoft.Search.Autopilot.NetMux">
+            <b:IPPrefix>{{ asic['ip6_prefix'] }}</b:IPPrefix>
+          </a:Prefix>
+          <a:PrefixStr>{{ asic['ip6_prefix'] }}</a:PrefixStr>
+        </a:LoopbackIPInterface>
+      </LoopbackIPInterfaces>
+      <ManagementIPInterfaces xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution">
+      </ManagementIPInterfaces>
+      <MplsInterfaces/>
+      <MplsTeInterfaces/>
+      <RsvpInterfaces/>
+      <Hostname>{{ asic['asicname'] }}</Hostname>
```

Comment:
The asic name for Fabric asic can be same as line card asic ?

Reply:
This block is supervisor only, do we need that check ?

Reply:
> The asic name for Fabric asic can be same as line card asic ?
When the ASIC are on individual cards, then the card's hostname helps to make it unique. 
When the ASICs local within a card, then they have to be unique

> This block is supervisor only, do we need that check ?
No, there can be a sku where there are multiple frontend asics and one or two backend asics.
IN that case we should not use type==supervisor check

Reply:
card_type is not defined or card_type != 'supervisor' will be required for the below loop to match what is done in minigraph_dpg.j2
```
{% for index in range(vms_number) %}
{% if vm_asic_ifnames[vms[index]][0].split('-')[1] == asic %}
```

Reply:
@saravanansv this comment is not addressed in the diff.